course_catalog/etl/video.py

```diff
@@ -2,65 +2,63 @@
 
 from django.conf import settings
 from googleapiclient.discovery import build
-from django.conf import settings
+import urllib3
+import yaml
+import certifi
+
+CONFIG_FILE = 'https://raw.githubusercontent.com/mitodl/open-video-data/master/youtube.yaml'
```

Comment:
Nit: `CONFIG_FILE_URL`, should also be defined in `settings.py` so we can override it without changing code